lib/galaxy/tool_util/parser/yaml.py

```diff
@@ -40,6 +40,12 @@ def parse_edam_operations(self):
     def parse_edam_topics(self):
         return self.root_dict.get("edam_topics", [])
 
+    def parse_xrefs(self):
+        xrefs = self.root_dict.get("xrefs", [])
+        if xrefs is None:
+            return []
```

Comment:
Lines 45-46 are not needed (`xrefs` will never be `None`).

Reply:
Why?